traefik/README.md

```diff
@@ -86,45 +139,77 @@ The [common Apache Access format][12] is used by default and is supported by thi
 
       Change the `path` and `service` parameter values and configure them for your environment.
 
-3. [Restart the Agent][9]
+4. [Restart the Agent][19]
+
+#### Trace collection
+
+**Available for Traefik v1.7+**
+
+1. [Enable APM][23] for Datadog, if needed.
+2. Follow [Traefix's documentation][24] to send traces to Datadog.
+
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
 
 ### Validation
 
-[Run the Agent's `status` subcommand][13] and look for `traefik` under the Checks section.
+[Run the Agent's `status` subcommand][25] and look for `traefik` under the Checks section.
 
 ## Compatibility
 
 The check is compatible with all major platforms.
 
+**Metrics**
```

Comment:
```suggestion
## Metrics
```

Reply:
Hey @apigirl ! I tried to use ##Metrics but then the section would disappear. I think it might be related to how the magic sentence is setup under ##Metrics, even though I didn't use the magic sentence under the new "Metrics" section that I added.